public/app/features/provisioning/ConfigForm.tsx

```diff
@@ -139,8 +140,8 @@ export function ConfigForm({ data }: ConfigFormProps) {
         <Input {...register('title', { required: 'This field is required.' })} placeholder={'My config'} />
       </Field>
       {type === 'github' && (
-        <FieldSet label="GitHub">
-          <Field label={'Token'} required error={errors?.token?.message} invalid={!!errors.token}>
+        <>
+          <Field label={'Github token'} required error={errors?.token?.message} invalid={!!errors.token}>
```

Comment:
```suggestion
          <Field label={'GitHub token'} required error={errors?.token?.message} invalid={!!errors.token}>
```
Let's make sure to spell `GitHub` correctly, especially in the user-facing parts.